addons/events/fnc_addBISPlayerEventHandler.sqf

```diff
@@ -0,0 +1,71 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_addBISPlayerEventHandler
+
+Description:
+    Adds an engine event handler just to the controlled entity.
+
+Parameters:
+    _eventType     - Type of event to add. Can be any engine (not mission) event. <STRING>
+    _eventCode     - Code to run when event is raised. <CODE>
+    _ignoreVirtual - Ignore virtual units (spectators, virtual zeus, UAV RC) [optional] (default: false) <BOOLEAN>
```

Comment:
Is including 'virtual entities' what the average user wants? If no, the default value should be `true`.